cf-agent/verify_packages.c

```diff
@@ -2734,14 +2737,15 @@ static bool ExecuteSchedule(EvalContext *ctx, const PackageManager *schedule, Pa
                     const Promise *const ppi = pi->pp;
                     Attributes a = GetPackageAttributes(ctx, ppi);
 
-                    char *offset = command_string + strlen(command_string);
+                    const size_t command_len = strlen(command_string);
+                    char *offset = command_string + command_len;
 
                     if ((a.packages.package_file_repositories) && ((action == PACKAGE_ACTION_ADD) || (action == PACKAGE_ACTION_UPDATE)))
                     {
                         const char *sp = PrefixLocalRepository(a.packages.package_file_repositories, pi->name);
                         if (sp != NULL)
                         {
-                            strcat(offset, sp);
+                            strlcat(offset, sp, estimated_size - command_len);
```

Comment:
Not your fault, but I hate that `offset` is a `char *` and not a offset (size_t). This would be much more readable, IMO:

```
strlcat(command_string + offset, sp, estimated_size - offset);
```